nova/templates/vspc-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nova-vmware-vspc-config
+  labels:
+    system: openstack
+    type: config
+    component: nova 
+data:
+  config.json: |
+   command : "vmware-vspc --config-file vspc.conf"
```

Comment:
Hi, now the config.json will only contain
`command : "vmware-vspc --config-file vspc.conf"`, which is not a proper json format.